api/sync15/blobdoc.go

```diff
@@ -109,6 +109,42 @@ func (d *BlobDoc) IndexReader() (io.ReadCloser, error) {
 	return pipeReader, nil
 }
 
+func (d *BlobDoc) ReadContentTags(fileEntry *Entry, r RemoteStorage) error {
+	if strings.HasSuffix(fileEntry.DocumentID, ".content") {
+		contentFile := archive.Content{}
+
+		meta, err := r.GetReader(fileEntry.Hash)
+		if err != nil {
+			return err
+		}
+		defer meta.Close()
+		content, err := ioutil.ReadAll(meta)
+		if err != nil {
+			return err
+		}
+		err = json.Unmarshal(content, &contentFile)
+		if err != nil {
+			log.Error.Printf("cannot read content %s %v", fileEntry.DocumentID, err)
```

Comment:
should we return the error here?

Reply:
Yes, I should have done that initially. Updated.